dm/dm/ctl/master/start_stop_validation.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/tiflow/dm/dm/config"
+	"github.com/pingcap/tiflow/dm/dm/ctl/common"
+	"github.com/pingcap/tiflow/dm/dm/pb"
+)
+
+func NewStartValidationCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start [-s source ...] [--mode mode] [--all-task] [task-name]",
+		Short: "start to validate the completeness of the data",
+		RunE:  startStopValidation("start"),
+	}
+	cmd.Flags().Bool("all-task", false, "whether applied to all tasks")
+	cmd.Flags().String("mode", "full", "specify the mode of validation: full, fast")
+	return cmd
+}
+
+func NewStopValidationCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stop [-s source ...] [--all-task] [task-name]",
+		Short: "stop validating the completeness of the data",
+		RunE:  startStopValidation("stop"),
+	}
+	cmd.Flags().Bool("all-task", false, "whether to all tasks")
+	return cmd
+}
+
+func startStopValidation(typ string) func(*cobra.Command, []string) error {
```

Comment:
```suggestion
func startStopValidation(op string) func(*cobra.Command, []string) error {
```

Reply:
> ignored + non-ignored = all

As the result I discussed with zhaoyang in the spec, all-errors = ignored errors + resolved-errors + current-error. I think all types of errors and the detailed functionality of this command should be definite before we resolve it😥 Maybe I should discuss it with PM again.